sources/packages/api/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt-request.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from "@nestjs/common";
+import { InjectLogger } from "../../common";
+import { LoggerService } from "../../logger/logger.service";
+import {
+  ConfigService,
+  DisbursementReceiptService,
+  ReportService,
+} from "../../services";
+import { ESDCFileHandler } from "../esdc-file-handler";
+import { DailyDisbursementUploadResult } from "./models/disbursement-receipt-integration.model";
+import { ReportsFilterModel } from "src/services/report/report.models";
+import { DisbursementReceiptIntegrationService } from "./disbursement-receipt-integration.service";
+
+@Injectable()
+export class DisbursementReceiptRequestService extends ESDCFileHandler {
+  constructor(
+    configService: ConfigService,
+    private readonly reportService: ReportService,
+    private readonly disbursementReceiptService: DisbursementReceiptService,
+    private readonly integrationService: DisbursementReceiptIntegrationService,
+  ) {
+    super(configService);
+  }
+
+  /**
+   * 1. Fetches the Daily disbursement information which are not sent.
```

Comment:
The number 1 does not seem to be accurate. It will fetch based on the `batchRunDate` or max date if `batchRunDate` is not provided, right?